### ISSUE TITLE
Dostęp do nazwy konta allegro

### ISSUE THREAD

**Issue by tomylee0** [allegro/allegro-api#9955]
Od sprzedającego dostajemy client id i client secret. Sprzedający łączy się z naszą aplikacją używając trybu DEVICE. Czy jest jakiś inny sposób automatycznego dostania się teraz do nazwy konta bez potrzeby nadawania przez sprzedającego uprawnienia "Dane osobowe" (allegro:api:profile:read)? Może jest jakoś nazwa konta ukryta w refresh/access tokenie? Lub może da się "podstępem" dostać mając uprawnienia do "Zarządzania ofertami"?

Pozdrawiam

**Comment by MaciejFrackowiak** [comment]
Nie, w tokenie zakodowane jest wyłącznie "user_name" w postaci identyfikatora. Login zwracamy w [GET /me](https://developer.allegro.pl/documentation/#operation/meGET), ale jak zauważyłeś potrzebny tam jest scope "allegro:api:profile:read".
